packages/cli-kit/src/public/node/path.ts

```diff
@@ -1,14 +1,31 @@
-import {OverloadParameters} from './typing/overloaded-parameters.js'
+import {OverloadParameters} from '../../typing/overloaded-parameters.js'
 import commondir from 'commondir'
-import {relative, dirname, join, normalize, resolve, basename, extname, isAbsolute, parse} from 'pathe'
+import {relative, dirname, join, normalize, resolve, basename, extname, isAbsolute} from 'pathe'
 import {findUp as internalFindUp} from 'find-up'
 import {fileURLToPath} from 'url'
 // eslint-disable-next-line node/prefer-global/url
 import type {URL} from 'url'
 import type {Pattern, Options} from 'fast-glob'
 
-export {join, relative, dirname, normalize, resolve, basename, extname, isAbsolute, parse}
+// Reexport methods from pathe
+export {
+  join as joinPath,
+  relative as relativePath,
+  normalize as normalizePath,
+  resolve as resolvePath,
+  isAbsolute as isAbsolutePath,
+  dirname,
+  basename,
+  extname,
+}
 
+/**
+ * Traverse the file system and return pathnames that match the given pattern.
+ *
+ * @param pattern - A glob pattern or an array of glob patterns.
+ * @param options - Options for the glob.
+ * @returns A promise that resolves to an array of pathnames that match the given pattern.
+ */
 export async function glob(pattern: Pattern | Pattern[], options?: Options): Promise<string[]> {
```

Comment:
Same here, I'd extract this one into `fs` because it does filesystem IO. We can do it in a separate PR.